app/src/main/res/layout/activity_main.xml

```diff
@@ -2,33 +2,51 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <data class="com.mycompany.myapp.ui.main.MainActivityBinding">
+    <data>
```

Comment:
Any reason for removing the explicit class binding class names? (I can go either way, just curious if there is a reason).

Reply:
In the newest android studio version it started highlighting it as an error. I'm not quite sure if it's because of navigation, but it is also just less code when letting the IDE generate the binding name itself, then it also forces all the files to have a specific naming convention from habit.